docs/utilities/worksheet_layout_utility.rst

```diff
@@ -0,0 +1,62 @@
+Utility for extending Worksheet layouts
+-----------------------------
+
+Use this tool if you want to create and use your own custom worksheet layout views.
+
+
+1. Create class which implements ``IWorksheetLayouts`` marker interface and returns the list of your homemade layouts (as tuple of tuples) 
+
+.. code-block:: python
+
+    from zope.interface import implements
+    from bika.lims.interfaces import IWorksheetLayouts
+
+    class MyClassForWorksheetLayouts(object):
+        """ My worksheet layouts
+        """
+        implements(IWorksheetLayouts)
+
+        def getLayoutsList(self):
+            return (
+                ('my_first_ws_layout', 'My first WS layout'),
+            )
+
+Please note that each tuple which describes your layout has two elements: (``registered_view_name``, ``Title``). Title used in the drop-down menu.
+
+
+2. Register your class as an utility in ``configure.zcml``
+
+.. code-block:: xml
+
+    <utility
+        name="my_name_worksheet_layouts_utility"
+        factory="my.module.some_module.MyClassForWorksheetLayouts"
+        provides="bika.lims.interfaces.IWorksheetLayouts"
+    />
```

Comment:
Optional:
I would rather implement it as a multi adapter for context and request instead of a utility.